custom/abdm/milestone_one/views/abha_verification_views.py

```diff
@@ -0,0 +1,68 @@
+from rest_framework.decorators import api_view, permission_classes, authentication_classes
+from rest_framework.permissions import IsAuthenticated
+from custom.abdm.auth import UserAuthentication
+
+from custom.abdm.milestone_one.utils import abha_verification_util as abdm_util
+from custom.abdm.milestone_one.utils.decorators import required_request_params
+from custom.abdm.milestone_one.utils.response_handler import get_response, generate_invalid_req_response
+
+
+@api_view(["GET"])
+@permission_classes((IsAuthenticated,))
+@authentication_classes((UserAuthentication,))
+def get_auth_methods(request):
+    health_id = request.query_params.get("health_id")
+    if not health_id:
+        error_msg = "Missing required parameter in the request param: health_id"
```

Comment:
could use `@required_request_params(["health_id"])`

Reply:
This is a GET request and required_request_params is for POST request params. Didn't add extra decorator/logic as this is the only GET request.